usage.txt

```diff
@@ -8,3 +8,4 @@ Usage
 
   Optional options:
     out:               path to output the installer (path must exist)
+    nsiTemplate:       path to override the installer.nsi template for win platform (path must exist)
```

Comment:
> the path would be relative to the place you run the command / script from then

It should handle relative and absolute paths. And currently it's supporting also both. So that shouldn't be a big deal :)